docs/getting-started.md

```diff
@@ -36,8 +45,16 @@ Run the gulp command in your project directory:
 gulp
 ```
 
+#### Result
+
 Voila! The default task will run and do nothing.
 
+```sh
+Using gulpfile ~/gulp/gulpfile.js
+[11:15:51] Starting 'default'...
+[11:15:51] Finished 'default' after 103 μs
+```
+
 To run multiple tasks, you can use `gulp <task> <othertask>`.
```

Comment:
No idea whether it helps or confuses, because I don't really know what it means.  Maybe a screenshot instead?  Or we could just delete it?  Do people know what it should look like when it "does nothing"?